docs/docs/ias.md

```diff
@@ -1,41 +1,49 @@
 # Intel Attestation Service
 
-!!! important
-    You don't need to read and perform these steps to deploy a Conclave app, unless you wish to use older
-    hardware that isn't capable of using the DCAP protocol. [Azure Gen 2 Virtual Machines](azure.md) in particular do not
-    need any of these steps.
+!!!Note
+    
+    You don't need to perform these steps if you use [Azure Gen 2 VMs](machine-setup.md) or hardware that supports the
+    DCAP attestation protocol.
```

Comment:
If we do not support DCAP it means that this is only for EPID.
https://api.portal.trustedservices.intel.com/EPID-attestation
 So I would remove the note ( I don't like notes/warnings, I really prefer to have the minimal amount of them) and change the title
`Intel Attestation Service (EPID attestation only)`.
We can explain things in the first paragraph here below (just an example, feel free to make it better)
`...security patches. Intel Attestation Service is only used in the context of EPID attestation protocol. DCAP, the more recent attestation protocol, follows a different attestation flow`

Reply:
Done.